lib/kramdown-rfc2629.rb

```diff
@@ -39,14 +39,20 @@ def initialize(*doc)
 
       XREF_START = /\{\{(.*?)\}\}/u
 
+      def self.split_xref(t)
+        label, href = t.split('=')
+        label = label.sub(/\A\d/) { "_#{$&}" } # can't start an IDREF with a number
+        label = label.sub('DOI.', 'DOI_').gsub('/', '_') # DOIs are gross
```

Comment:
Why get rid of the dot?  It doesn't seem to be a problem with I-D.foo and other references.